doc/howto/network_acls.md

```diff
@@ -739,16 +840,16 @@ lxc network set <network_name> security.acls="<ACL_name>[,<ACL_NAME>,...]"
 
 #### Examples
 
-Set the `br01` network's `security.acls` to contain only the `web-traffic` ACL:
+Set the `ovn1` network's `security.acls` to contain only the `web-traffic` ACL:
```

Comment:
as this section is network type agnostic (the header says "bridge or OVN network"),  how about we just use the network name "my-network" here?